frontend/src/components/activity/ScheduleEntry.vue

```diff
@@ -348,6 +370,27 @@ export default {
         this.editActivityTitle = true
       }
     },
+    async copyUrlToClipboard() {
+      try {
+        const res = await navigator.permissions.query({ name: 'clipboard-read' })
+        if (res.state == 'prompt') {
+          this.$refs.copyInfoDialog.open()
+        }
+      } catch {
+        console.warn('clipboard permission not requestable')
+      }
+
+      const scheduleEntry = scheduleEntryRoute(this.scheduleEntry())
+      const url = window.location.origin + router.resolve(scheduleEntry).href
+      navigator.clipboard.writeText(url)
```

Comment:
This ensures the toast is only shown after successfully copying the text.
```suggestion
      await navigator.clipboard.writeText(url)
```
I think this might fix the first case I pointed out.